src/taipy/gui/utils/_variable_directory.py

```diff
@@ -103,7 +103,8 @@ def get_var(self, name: str, module: str) -> t.Optional[str]:
 
 
 _MODULE_NAME_MAP: t.List[str] = []
-_RE_TPMDL_DECODE = re.compile(r"(.*?)_TPMDL_(.*)")
+_RE_HAS_TPMDL = re.compile(r".*_TPMDL_.*")
```

Comment:
I don't see _RE_HAS_TPMDL anywhere... and the RE seems overkill.
If you need to find the string there are more effective ways

Reply:
it is used in Gui but you're right it is an overkill